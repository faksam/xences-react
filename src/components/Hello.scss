@import 'sass-rem';

body {
  display: block;
}
