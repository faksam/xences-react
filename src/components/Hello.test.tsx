// react libraries
import * as React from 'react';

// third-party libraries
import { shallow } from 'enzyme';

// components
import { Hello } from './Hello';

describe('Hello component', () => {
  const props = {
    compiler: '',
    framework: '',
  };

  it('should be rendered properly', () => {
    const wrapper = shallow(<Hello { ...props }/>);

    expect(wrapper).toMatchSnapshot();
  });
});
