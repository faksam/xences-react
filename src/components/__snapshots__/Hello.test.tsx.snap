// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello component should be rendered properly 1`] = `
<h1>
  Hello from 
   and 
  !
</h1>
`;
